lib/visualstudio.js

```diff
@@ -156,6 +156,8 @@ function detect(options, callback) {
 						appc.subprocess.getRealName(info.vcvarsall, function (err, vcvarsall) {
 							if (!err) {
 								info.vcvarsall = vcvarsall;
+								// vcvarsall may contain space
+								vcvarsall = vcvarsall.replace(/\ /g, '^ ');
```

Comment:
@sgtcoolguy yeah, I didn't know that but I saw it in Titanium CLI source. I'm searching where...